docs/gettinginvolved/functional-testing.rst

```diff
@@ -4,19 +4,23 @@ Functional Testing
 
 The tests for the various cloud block device backends depend on access to credentials supplied from the environment.
 
-The tests look for two environment variables:
-  ..
+The tests look for the following environment variables:
+
+.. XXX
      # FLOC-2090 This is yet another configuration file.
      # Make it just be the same as the acceptance testing configuration file.
 
-- ``FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_FILE``: This points at a YAML file with the credentials.
-- ``FLOCKER_FUNCTIONAL_TEST_CLOUD_PROVIDER``: This is the name of a top-level key in the configuration file.
+- ``FLOCKER_FUNCTIONAL_TEST``: This variable must be set.
+- ``FLOCKER_FUNCTIONAL_TEST_CLOUD_CONFIG_FILE``: This variable points at a YAML file with the credentials.
+- ``FLOCKER_FUNCTIONAL_TEST_CLOUD_PROVIDER``: This variable must be the name of a top-level key in the configuration file.
+- ``FLOCKER_FUNCTIONAL_TEST_AWS_AVAILABILITY_ZONE`` (AWS only): The AWS backend also requires that the availability zone that the test is running in to be specified.
```

Comment:
There's a similar variable for OpenStack, `FLOCKER_FUNCTIONAL_TEST_OPENSTACK_REGION` I think.